dev-python/google-auth-oauthlib/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
```

Comment:
Don't add new random ebuilds to be maintained by the python project, but you will need to be the maintainer if you wish it to be in gentoo tree. Via proxy-maintainer project of course.